src/go/migration-cli/tests/data/complex_cluster_001.yaml

```diff
@@ -0,0 +1,151 @@
+    redpanda.cluster_id: rp-chhnf2abksi1fj6tastg
+    redpanda.compacted_log_segment_size: "67108864"
+    redpanda.default_topic_replications: "3"
+    redpanda.enable_rack_awareness: "true"
+    redpanda.group_topic_partitions: "16"
+    redpanda.internal_topic_replication_factor: "3"
+    redpanda.kafka_batch_max_bytes: "1048576"
+    redpanda.kafka_connection_rate_limit: "1000"
+    redpanda.kafka_connections_max: "15100"
+    redpanda.log_segment_size: "134217728"
+    redpanda.log_segment_size_max: "268435456"
+    redpanda.log_segment_size_min: "16777216"
+    redpanda.max_compacted_log_segment_size: "536870912"
+    redpanda.max_concurrent_producer_ids: "25000"
+    redpanda.partition_autobalancing_mode: continuous
+    redpanda.topic_partitions_per_shard: "1000"
+  cloudStorage:
+    apiEndpoint: storage.googleapis.com
+    bucket: redpanda-cloud-storage-someblah
+    cacheStorage:
+      capacity: 450Gi
+      storageClassName: local-path
+    credentialsSource: gcp_instance_metadata
+    enabled: true
+    region: asia-south1
+    secretKeyRef: {}
+    trustfile: /etc/ssl/certs/ca-certificates.crt
+  configuration:
+    additionalCommandlineArguments:
+      abort-on-seastar-bad-alloc: ""
+      dump-memory-diagnostics-on-alloc-failure-kind: all
+    adminApi:
+      - external: {}
+        port: 9644
+        tls: {}
+    kafkaApi:
+      - authenticationMethod: sasl
+        external: {}
+        port: 9092
+        tls: {}
+      - authenticationMethod: sasl
+        external:
+          enabled: true
+          endpointTemplate: '{{ .Index }}-0ec4922b-{{ .HostIP | sha256sum | substr 0
+          7 }}'
+          subdomain: someblah.byoc.prd.cloud.redpanda.com
+        port: 30092
+        tls:
+          enabled: true
+          issuerRef:
+            kind: ClusterIssuer
+            name: letsencrypt-dns-prod
+    pandaproxyApi:
+      - external: {}
+        port: 8082
+        tls: {}
+      - authenticationMethod: http_basic
+        external:
+          enabled: true
+          endpointTemplate: '{{ .Index }}-0ff70780-{{ .HostIP | sha256sum | substr 0
+          7 }}'
+          ingress:
+            enabled: false
+          subdomain: someblah.byoc.prd.cloud.redpanda.com
+        port: 30082
+        tls:
+          enabled: true
+          nodeSecretRef:
+            apiVersion: v1
+            kind: Secret
+            name: rp-someblah-redpanda
+            namespace: redpanda
+    rpcServer:
+      port: 33145
+    schemaRegistry:
+      authenticationMethod: http_basic
+      external:
+        enabled: true
+        endpoint: schema-registry-b822f753
+        staticNodePort: true
+        subdomain: someblah.byoc.prd.cloud.redpanda.com
+      port: 30081
+      tls:
+        enabled: true
+        nodeSecretRef:
+          apiVersion: v1
+          kind: Secret
+          name: rp-someblah-redpanda
+          namespace: redpanda
+  image: docker.io/redpandadata/redpanda
+  initialValidationForVolume: true
```

Comment:
NIT: This should be implemented in helm chart. Init container should validate that mounted volume has file system of type XFS.

Reply:
We should probably file an issue then right?

Reply:
yes.

Reply:
No need. Redpanda will log that information.

Reply:
This wasn't the point of this option. In GCP when VM is recreated alongside with it's local disks our automation might hit a race condition problem within one Kubernetes Node where:

- one Redpanda is lost - VM recreation
- second Redpanda boots up with ex4 fs - It is scheduled on the same VM, but PV unfortunately will recreate path on disk using ex4 fs 
- third Redpanda will correctly boot with XFS - our automation format disks and creates RAID 0 array

In 3 node cluster we would have 2 nodes dead and 2 others Redpanda couldn't elect a leader, so that third Redpanda can not join this cluster and it is impossible to decommission 2 dead nodes. 

To conclude printing that information is not enough. Something must prevent Redpanda to start.